src/openforms/formio/rendering/tests/test_vanilla_formio_components.py

```diff
@@ -340,7 +340,7 @@ def test_wysiwyg_component(self):
             RenderModes.confirmation_email: False,
             RenderModes.pdf: True,
             RenderModes.cli: True,
-            RenderModes.summary: False,
+            RenderModes.summary: True,
```

Comment:
Is this right? I don't see any hint that `showInSummary` is enabled for this component, so then I'd expect it to *not* be displayed in the summary unless it's explicitly opted-in to?

I am looking at `RENDER_CONFIGURATION` in `src/openforms/formio/rendering/conf.py`, which basically states that if the option is not explicitly provided, then the default is to display it:

```py
    RenderModes.summary: RenderConfiguration(
        RenderConfigurationOptions.show_in_summary, default=True
    ),
```

However, our desired default behaviour for content component is that it is not displayed unless explicitly enabled (as per the checkbox), which is the inverse.

So I'd expect these tests:

* content component with explicit opt-out (`"showInSummary": False`) -> not visible
* content component with explicit opt-in (`"showInSummary": True`) -> visible
* content component with implicit opt-out (`showInSummary` absent) -> not visible

I'm not sure how realistic the missing key variant is - when using the form.io builder it does seem to always include the key with its default value. So maybe the tests just need to be updated to not have implicit/missing keys, as that doesn't reflect real life?

@SilviaAmAm please double check if my interpretation of the ticket/beviour is right

Reply:
Yes, I also thought this. I had suggested above that to make sure the default is included a migration should be added (then there shouldn't be content components without the `showInSummary` key). And then for the case where there is no key (although it shouldn't happen) use the same hardcoded default as other components specified in `RENDER_CONFIGURATION`

Reply:
okay, so then all that is missing is basically just the on/off cases in tests, as we now only have the edge case that shouldn't happen anyway because of the data migration, right?

Reply:
Ahh yes, indeed

Reply:
@pi-sigma can you please update these tests so we can merge this?

Reply:
Changes had been made already, but I forgot to push the latest code. Sorry about that!